src/services/accountService.ts

```diff
@@ -11,6 +11,10 @@ export async function getAccount(network: string, address: string) {
 	// if the address is encoded, decode it
 	const decodedAddress = decodeAddress(address);
 
+	if(decodedAddress.length !== 66) {
```

Comment:
Hele tohle neni ok. Odkud ze vlastne beres ze ta adresa ma byt dlouha 66 znaku? Co EVM adresy apod?

Reply:
Nevymyslej tu zadnou logiku, `isAddress` uz to checklo jesli je validni.